docs/tutorials/kubectl/app-management-overview.md

```diff
@@ -0,0 +1,238 @@
+
+Advantages over *Yaml Config with `apply`*
+
+- Behavior is simpler and easier to understand
+- Better support for ThirdPartyResources
+- Fewer edge cases
+
+Disadvantages over *Yaml Config with `apply`*
+
+- Works best on files, since it requires specify the operation (create, replace, delete)
+  - Cannot mix create, replace, delete for different resources
+- Updates to live resources must be reflected in config or will be lost during next replace.
+  - **If the resource spec in the cluster is written to by sources without updating the yaml config, those changes will be lost when replacing the resource spec.**
+
+<!---
+For a tutorial on how to use Yaml Config for app management, see:
+[App Management Yaml Config](/docs/tutorials/kubectl/app-management-using-yaml-config/)
+-->
+
+# Yaml Config with `apply`
+
+## Apply Example
+
+```sh
+kubectl apply -f configs/
+```
+
+## Apply Overview
+
+*Kubectl* has a declarative config management mode called `apply`.  This
+supports several capabilities that the basic yaml config
+do not support.
+
+Advantages over basic *Yaml Config*:
+
+- Retains changes made directly to the live resource even if they are not written to the backing config.
+- Operates on directories and automatically detects the operation type per-resources *(create, patch, delete)*
+
+Disadvantages over basic *Yaml Config*:
+
+- Working on directories and doing patches is more complicated
+- Still under development.  Known issues with specific resource types exist.
+- Limited support for ThirdPartyResources
+
+## Keeping changes without updating the config
+
+This is done by applying **only** the deltas between the new config and the last config
+to the live resource.
+
+An example of this use case would be when using a *HorizonalPodAutoscaler*
+to write the controller replica count, and omitting the replica count
+from the spec.
+
+## Operating on directories instead of files
+
+Apply can take a directory as the `-f` argument, and recursively apply
+each resource.  This will automatically detect whether a resource
+is being created or patched (note - this is different than replace).
+Detecting resources to delete is also supported, but in alpha only.
+
+## Important Considerations
+
+Apply updates resources by patching in changes to individual fields instead
+or by replacing the entire configuration.  This can make understanding
+the results of `apply` more challenging.
+
+<!---
+For a tutorial on how to use Yaml Config with multiple writers, see:
+[App Management Yaml Config](/docs/tutorials/kubectl/app-management-using-yaml-config-multiple-writers/)
+-->
+
+# Migrating a Resource From One Management Style to Another
+
+## From *Imperative Commands* -> *Yaml Config*
+
+1. Output the config for the resource to a file using:
+
+   ```sh
+   kubectl get type/name -o yaml > type_name.yaml
+   ```
+
+2. Delete the *Status* field from the config, since this is managed by the system
```

Comment:
I delete this section for now.  It deserves more attention than I can give it right now, and I don't want to hold up this PR just for this section.